GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -79,49 +79,29 @@ public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath
                 };
         }
 
-        public static ArgumentString GetRefsCmd(GetRefsEnum getRef, bool noLocks, GitRefsSortBy sortBy, GitRefsSortOrder sortOrder)
+        public static ArgumentString GetRefsCmd(GitRefs getRef, bool noLocks, GitRefsSortBy sortBy, GitRefsSortOrder sortOrder)
         {
-            string format;
-            bool hasTags = (getRef & GetRefsEnum.All) != 0 || (getRef & GetRefsEnum.Tags) != 0;
-            if (!hasTags)
-            {
-                // If we don't need tags, it is easy.
-                format = @"--format=""%(objectname) %(refname)""";
-            }
-            else
-            {
-                // ...however, if we're interested in tags, tags may be simple (in which case they are point to commits directly),
-                // or "dereferences" (i.e. commits that contian metadata and point to other commits, "^{}").
-                // Derefence commits do not contain date information, so we need to find information from the referenced commits (those with '*').
-                // So the following format is as follows:
-                //      If (there is a 'authordate' information, then this is a simple tag/direct commit)
-                //      Then
-                //          format = %(objectname) %(refname)
-                //      Else
-                //          format = %(*objectname) %(*refname) // i.e. info from a referenced commit
-                //      End
-                format = @"--format=""%(if)%(authordate)%(then)%(objectname) %(refname)%(else)%(*objectname) %(*refname)%(end)""";
-            }
+            bool hasTags = (getRef & GitRefs.All) != 0 || (getRef & GitRefs.Tags) != 0;
 
             GitArgumentBuilder cmd = new("for-each-ref",
                 gitOptions: noLocks && GitVersion.Current.SupportNoOptionalLocks
                     ? (ArgumentString)"--no-optional-locks"
                     : default)
             {
-                { sortBy != GitRefsSortBy.Default, GetSortCriteria(hasTags, sortBy, sortOrder), string.Empty },
-                format,
-                { HasFlag(getRef, GetRefsEnum.Branches), "refs/heads/", string.Empty },
-                { HasFlag(getRef, GetRefsEnum.Remotes), "refs/remotes/", string.Empty },
-                { HasFlag(getRef, GetRefsEnum.Tags), "refs/tags/", string.Empty },
+                sortCriteria(hasTags, sortBy, sortOrder),
+                gitRefsFormat(getRef, hasTags),
+                gitRefsPattern(getRef)
```

Comment:
```suggestion
                GetSortCriteria(hasTags, sortBy, sortOrder),
                GetGitRefsFormat(getRef, hasTags),
                GetGitRefsPattern(getRef)
```
?
Or else new convention for local functions?

Reply:
I had a squiggle to adjust the name that I acted on but do not see that now, changing back...